Makefile

```diff
@@ -62,7 +62,7 @@ release_r: CLIENT_FLAGS=-DBUILD_R=1
 release_r: release
 
 release_python: CLIENT_FLAGS=-DBUILD_PYTHON=1 -DBUILD_JSON_EXTENSION=1 -DBUILD_FTS_EXTENSION=1 -DBUILD_TPCH_EXTENSION=1 -DBUILD_VISUALIZER_EXTENSION=1 -DBUILD_TPCDS_EXTENSION=1
-release_python: debug
+release_python: release
```

Comment:
Slightly unrelated, but could you also push these to the https://github.com/duckdb/extension-template makefile? :-)

Reply:
Sure, no problem!